ui/src/app/views/account-creation/component.tsx

```diff
@@ -78,30 +79,24 @@ export class AccountCreation extends
 
   createAccount(): void {
     const {invitationKey, setProfile} = this.props;
+    const profile = this.state.profile;
+    const {givenName, familyName, username, contactEmail,
+      currentPosition, organization, areaOfResearch} = this.state.profile;
```

Comment:
nit: `profile`. Also arguably I'm not sure I would redeclare these as they're only used once `profile.givenName` etc seems fine.